secure_fw/partitions/crypto/crypto_check_config.h

```diff
@@ -9,62 +9,113 @@
 
 #include "config_tfm.h"
 
-#if CRYPTO_RNG_MODULE_ENABLED  && \
-    (!defined(MBEDTLS_CTR_DRBG_C) &&            \
-     !defined(MBEDTLS_HMAC_DRBG_C) &&           \
-     !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG))
-#error "CRYPTO_RNG_MODULE_ENABLED enables, but not all prerequisites (missing RNG)!"
+/* RNG module config can't be adjusted on PSA_WANT_* requirements
+ * because the PSA Crypto subsystem guarantees on booting that a
+ * source of entropy with a configured deterministic random number
```

Comment:
I would propose to change the wording so that it doesn't say deterministic random number generator. 
Probably better to replace it with something like: cryptographically secure pseudorandom number generator

After all it is definitely not  deterministic.

Reply:
I just removed all of that. Not important information.